datastore/cloud-client/snippets.py

```diff
@@ -887,6 +913,9 @@ def property_by_kind_run_query(client):
 
 def regional_endpoint():
     # [START datastore_regional_endpoints]
+    from google.cloud import datastore
```

Comment:
Why is this the only sample that has this import?

Reply:
All the other ones also include [the preamble](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/ec9128da73d7832cb970cd0442562a54be1a1ad1/datastore/cloud-client/snippets.py#L17), which imports datastore and creates the client